spec/acceptance/puppetserver_upgrade_spec.rb

```diff
@@ -17,10 +17,10 @@
     case ENV['BEAKER_PUPPET_COLLECTION']
     when 'puppet8'
       from_version = '8.2.0'
-      to_version = '8.3.0'
+      to_version = '8.5.0'
```

Comment:
this one is (obviously) not necessary, but I thought if we're already bumping ranges, we can go all the way :)